### DISCUSSION TITLE
Is this dead/over?

### DISCUSSION THREAD

**Discussion by R3Cardoso** [basnijholt/adaptive-lighting#850] [General]
Is the project Adapative Lighting over?
I see there is no updates since August and no answers on issues to anyone.

**Comment by basnijholt** [comment]
No. Have 2 babies and just started a new job. Feel free to make a PR to fix things 😉

**Comment by R3Cardoso** [comment]
No problem, i got two childs too.
Was just to know. I love this integration, I can't have my lights setup with them anymore.
